meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -528,13 +528,19 @@ if [ "$LUKS" = "1" ]; then
     umount "$EFI_MOUNT_DIR"
 fi
 
+inform "Board specific flash procedure..."
+/usr/bin/resin-init-flasher-board
+
+info "Log end"
+# Preserve logs when running from initramfs
+if [ -n "${INITRAMFS_LOGFILE}" ] && [ -f "${INITRAMFS_LOGFILE}" ]; then
+    cp "${INITRAMFS_LOGFILE}" "${INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT}/migration_$(date +"%Y%m%dT%H%M")"
```

Comment:
Any chance of this filling up the boot partition if many migrations are attempted?

Reply:
Each migration is a complete re-flash, so no, there will only be at most one migration present on disk.